### DISCUSSION TITLE
Prisma Pulse — Real-time database events with type-safe subscriptions

### DISCUSSION THREAD

**Discussion by nikolasburk** [prisma/prisma#19731] [Announcements]
Hey everyone 👋 

We are excited to release our newest product [**Pulse**](http://prisma.io/data-platform/pulse?utm_source=github&utm_medium=pinned-discussion)! 

It is a managed [Change Data Capture](https://en.wikipedia.org/wiki/Change_data_capture) (CDC) service that allows you to subscribe to real-time events in your database via Prisma Client. 

**Pulse is the perfect tool if you need any kind of real-time functionality in your app!**

Here's a sample code snippet that sends updates to your application when something changes on the `message` table in your database:

```ts
// Subscribe to new events on the `message` table
const liveQuery = prisma.message.subscribe()
    
// Waiting loop that prints new events when something changes in the database
for await (const event of liveQuery) {
  console.log(event.action); // 'create', 'update', 'delete'
}
```

You can learn more about Pulse in the [announcement article](https://www.prisma.io/blog/introducing-pulse-jtu4UPC8ujy4) and [get started with it today](https://console.prisma.io/login?utm_source=github&utm_medium=pinned-discussion-pulse).

**Comment by divait** [comment]
What happens with Prisma Pulse?

**Comment by drakedeatonuk** [comment]
Is this service still available?
